src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IReleaseFilesService.cs

```diff
@@ -11,9 +11,9 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces
     public interface IReleaseFilesService
     {
         Task<Either<ActionResult, DataFileInfo>> UploadDataFilesAsync(Guid releaseId,
-            IFormFile dataFile, IFormFile metaFile, string name, string userName);
+            IFormFile dataFile, IFormFile metaFile, string userName, string subjectName = null, Guid? replacingId = null);
 
-        Task<Either<ActionResult, DataFileInfo>>  UploadDataFilesAsZipAsync(Guid releaseId, IFormFile zipFile, string name, string userEmail);
+        Task<Either<ActionResult, DataFileInfo>>  UploadDataFilesAsZipAsync(Guid releaseId, IFormFile zipFile, string subjectName, string userEmail);
```

Comment:
Probably worth changing these method names to drop the `Async` postfix

Reply:
Thanks, I've done this now. I did the others too.